resen/resencmd.py

```diff
@@ -40,16 +42,66 @@ def do_create_bucket(self,args):
         # check if bucket_name has spaces in it and is greater than 20 characters
         # also bucket name must start with a letter
         if ' ' in bucket_name or len(bucket_name) > 20 or not bucket_name[0].isalpha():
-            print("Syntax Error")
+            print("Syntax Error. Usage: create_bucket bucket_name")
             return
+
+        print("Creating bucket with name: %s" % bucket_name)
         status = self.program.create_bucket(bucket_name)
 
+        # ask user about resen-core version
+        valid_versions = sorted([x['version'] for x in self.program.bucket_manager.valid_cores])
```

Comment:
If the bucket name already exists, then we could use `status`, that is `False` to stop the `create_bucket` workflow.
Something like:
```
if not status:
    print("You can use start_bucket if the bucket is not running and start_jupyter instead.")
    return
```
or just return

Reply:
How we handle the status is a bigger problem to deal with than the scope of this PR.

Note that the call to `self.program.create_bucket` already spits out an error. The solution is to have the `BucketManager` class raise exceptions and then in `resencmd.py` we would catch and handle those.

Reply:
So what about:
```
if not status:
    return
```

Reply:
Now that I've had coffee, I understand that what you are meaning to say is that if the bucket creation barfs at any point, we get a "partially created" bucket at best and at worst an error occurs that crashes the `create_bucket` command and the resen shell completely.

Yes, for now I'll need to add something that deals with this.